### PR TITLE
Hamid914 add hometown

### PR DESCRIPTION
**Pull request by hamid914**
Anyone that's been to Shahrekord -- have you done any local stuff you enjoy? 🌴

@crichid What kinds of things would you like to do if you ever visit Shahrekord? 🐭

### CODE REVIEW

File: hamid914-shahrekord.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Shahrekord, Chaharmahal-O-Bakhtiari, Iran
+
+### Places to Eat
+- Mr. Dizi
```
Comment (ultramag) [inline hamid914-shahrekord.md]: any place else to eat?

Comment (lucy-sheridan) [inline hamid914-shahrekord.md]: wewewew

**Comment by lucy-sheridan** [comment]
dfdsf

**Comment by hamid914** [comment]
@ultramag I added another one 😋

**Pull request closed without merging**